package.json

```diff
@@ -65,7 +65,7 @@
     "eslint": "^4.13.1",
     "eslint-config-seek": "^3.0.0",
     "express": "^4.16.2",
-    "extract-text-webpack-plugin": "^3.0.1",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
Should we wait for v4.0.0 final before merging this?

Reply:
Yes, I think we should

Reply:
Looks like we'll need to migrate to [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) instead: https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/755#issuecomment-374548202

Reply:
I see. Sounds good